projects/plugins/jetpack/modules/masterbar/admin-menu/menu-mappings.php

```diff
@@ -23,6 +23,20 @@
 	'edit.php?post_type=jetpack-testimonial' => 'https://wordpress.com/types/jetpack-testimonial/',
 );
 
+if (
+	/**
+	 * Filter to enable the modernized Reading Settings in Calypso UI.
+	 *
+	 * @since $$next-version$$
+	 * @module masterbar
+	 *
+	 * @param bool false Should the modernized Reading Settings be enabled? Default to false.
+	 */
```

Comment:
We do not need a full docblock here; we do not want to create 2 Codex pages about the same hook.

```suggestion
	 /** This filter is documented in modules/masterbar/admin-menu/class-admin-menu.php */
```

Reply:
Thank you, Jeremy!